README.md

```diff
@@ -9,6 +9,7 @@ This is the Ekino Wordpress plugin to interact with Symfony.
 1) Put the following lines at the beginning of your `wp-config.php` file, according to your project configuration:
 
 ```php
+define('WP_SYMFONY_VERSION', 3.0);
```

Comment:
There is no need to a constant. Just use the constant version defined in the Kernel class.